apps/HelloBaremetal/.gitignore

```diff
@@ -0,0 +1 @@
+!enable_neon.s
```

Comment:
@steven-johnson - we should add some sort of linter / autoformatter to prevent line-ending churn